config/routes.rb

```diff
@@ -3,4 +3,8 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
+  resources :mechanics, only: [:show, :update]
```

Comment:
Nice job using `resources`. This would change when you make a `create` vs a `update`. You would probably hand roll your `post` route.